### PR TITLE
[5780] Projects - don't cancel the marketplace/search request when loading the page.

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/5780

### CODE REVIEW

File: ui/app/src/components/CreateSiteDialog/CreateSiteDialogContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,616 @@
...
+import EmptyState from '../EmptyState';
+import BlueprintForm from './BlueprintForm';
+import BlueprintReview from './BlueprintReview';
+import Spinner from '../Spinner';
+import DialogFooter from '../DialogFooter';
+import PrimaryButton from '../PrimaryButton';
+import { useStyles } from './styles';
+import messages from './translations';
+
+// TODO: enhance this props
```
Comment (rart) [inline ui/app/src/components/CreateSiteDialog/CreateSiteDialogContainer.tsx]: ?

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/CreateSiteDialog/CreateSiteDialogContainer.tsx (hunk 1 above)
Comment (jvega190) [reply ui/app/src/components/CreateSiteDialog/CreateSiteDialogContainer.tsx]: Forgot to remote the TODO comment

File: ui/app/src/components/CreateSiteDialog/CreateSiteDialogContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,615 @@
...
+  };
+  setSearch(search): void;
+  handleClose(event?, reason?): void;
+  dialog: {
+    open: boolean;
+    inProgress: boolean;
+  };
+  setDialog(dialog): void;
+  disableEnforceFocus: boolean;
+}
```
Comment (rart) [inline ui/app/src/components/CreateSiteDialog/CreateSiteDialogContainer.tsx]: You removed the TODO but didn't actually fix it.

**Review by rart — CHANGES_REQUESTED** [review]

File: ui/app/src/components/CreateSiteDialog/CreateSiteDialogContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,619 @@
...
+
+interface DialogState {
+  open: boolean;
+  inProgress: boolean;
+}
+
+interface CreateSiteDialogContainerProps {
+  site: SiteState;
+  setSite(site): void;
```
Comment (rart) [inline ui/app/src/components/CreateSiteDialog/CreateSiteDialogContainer.tsx]: Type missing

**Review by rart — CHANGES_REQUESTED** [review]

**Review by rart — APPROVED** [review]

**Pull request merged**